### PR TITLE
Document single-element tuple support for the remaining pooling ops

### PR DESCRIPTION
**Pull request by LuShadowX**
Fixes the remaining part of #153149.

#154353 updated `F.avg_pool2d` to document that `kernel_size`, `stride` and `padding` accept a single-element tuple, but the sibling pooling ops kept the older wording that mentions only a single number or a full-length tuple. ATen has always accepted the size-1 form — each op checks `kernel_size.size() == 1 || kernel_size.size() == N` and reuses element 0 for the remaining spatial dimensions (e.g. `aten/src/ATen/native/AveragePool3d.cpp:30-34`) — so only the documentation was wrong, which @albanD confirmed on the issue.

Reworded to match the existing `avg_pool2d` text: `F.avg_pool3d`, `F.max_pool2d`, `F.max_pool3d`, and the `nn.AvgPool3d`, `nn.MaxPool2d`, `nn.MaxPool3d` class docstrings. The 1D ops need no change — for them `(kW,)` is already the documented tuple form.

## Test Plan

Docs-only change, so the check is that the newly documented behavior is real. Every parameter the reworded text covers accepts a single-element tuple and returns a bit-identical result to the scalar form:

```python
import torch, torch.nn.functional as F
x2 = torch.rand(1, 3, 8, 8); x3 = torch.rand(1, 3, 8, 8, 8)
rows = [
 ("F.avg_pool3d", lambda: F.avg_pool3d(x3, (3,), stride=(2,), padding=(1,)),
                  lambda: F.avg_pool3d(x3,  3,  stride= 2,  padding= 1)),
 ("F.max_pool2d", lambda: F.max_pool2d(x2, (3,), stride=(2,), padding=(1,), dilation=(1,)),
                  lambda: F.max_pool2d(x2,  3,  stride= 2,  padding= 1,  dilation= 1)),
 ("F.max_pool3d", lambda: F.max_pool3d(x3, (3,), stride=(2,), padding=(1,), dilation=(1,)),
                  lambda: F.max_pool3d(x3,  3,  stride= 2,  padding= 1,  dilation= 1)),
 ("nn.AvgPool3d", lambda: torch.nn.AvgPool3d((3,), stride=(2,), padding=(1,))(x3),
                  lambda: torch.nn.AvgPool3d( 3,  stride= 2,  padding= 1)(x3)),
 ("nn.MaxPool2d", lambda: torch.nn.MaxPool2d((3,), stride=(2,), padding=(1,), dilation=(1,))(x2),
                  lambda: torch.nn.MaxPool2d( 3,  stride= 2,  padding= 1,  dilation= 1)(x2)),
 ("nn.MaxPool3d", lambda: torch.nn.MaxPool3d((3,), stride=(2,), padding=(1,), dilation=(1,))(x3),
                  lambda: torch.nn.MaxPool3d( 3,  stride= 2,  padding= 1,  dilation= 1)(x3)),
]
for name, tup, sca in rows:
    a, b = tup(), sca()
    print(f"{name:14} {str(tuple(a.shape)):22} {str(tuple(b.shape)):22} {torch.equal(a, b)}")
```

```
F.avg_pool3d   (1, 3, 4, 4, 4)        (1, 3, 4, 4, 4)        True
F.max_pool2d   (1, 3, 4, 4)           (1, 3, 4, 4)           True
F.max_pool3d   (1, 3, 4, 4, 4)        (1, 3, 4, 4, 4)        True
nn.AvgPool3d   (1, 3, 4, 4, 4)        (1, 3, 4, 4, 4)        True
nn.MaxPool2d   (1, 3, 4, 4)           (1, 3, 4, 4)           True
nn.MaxPool3d   (1, 3, 4, 4, 4)        (1, 3, 4, 4, 4)        True
```

Lint:

```
lintrunner --take RUFF,PYFMT torch/nn/functional.py torch/nn/modules/pooling.py
```

```
ok No lint issues.
```

Two related things I left out of this PR to keep it to one idea: `AvgPool3d`'s parameter block lists only `kernel_size` and `stride` even though `padding` also accepts both forms, and `F.lp_pool2d` rejects a single-element tuple unlike every other pooling op. Happy to follow up on either.

This PR was prepared with the assistance of an AI coding assistant; I have reviewed the change and the verification above.

### CODE REVIEW

**Review by Mr-Ashish — CHANGES_REQUESTED** [review]
## 🏴‍☠️ Torii — REQUEST_CHANGES
**Score:** 69/100 · **Confidence:** medium

Torii: REQUEST CHANGES

_Full review is the PR comment with marker `<!-- torii-review pr=191857`._
